src/content/docs/en/reference/modules/astro-i18n.mdx

```diff
@@ -337,3 +341,116 @@ export const onRequest = defineMiddleware(async (context, next) => {
   return new Response("Not found", { status: 404 });
 })
 ```
+
+### `normalizeTheLocale()`
+
+<p>
+
+**Type:** `(locale: string) => string`
```

Comment:
Do we need a `<Since />` here? Maybe it makes sense to use the default `4.0.0` rather than nothing?